qiskit/circuit/library/__init__.py

```diff
@@ -492,6 +492,7 @@
 from .blueprintcircuit import BlueprintCircuit
 from .generalized_gates import (
     Diagonal,
+    _dict_to_list,
```

Comment:
Why do we want to import this private function up into a very high-level namespace?